library/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,9 @@
 	<color name="transparent">#00000000</color>
 	<color name="borderGrey">#c2c2c2</color>
 	<color name="spotlightIndicatorLightGrey">#aeaeae</color>
+	<color name="headingText">#6d6d6d</color>
```

Comment:
I would have used the name of the colour, instead of the object is used, e.g. if you want to use the same colour on a different place you do not need to create new or use something that will look "weird" e.g(ProductItem with colour badgeBorder)

Reply:
Ahh yeah that's a good point. Makes more sense than my way!

Reply:
For a normal project that makes sense, but for the libraries, I give everything it's own colour object, so that it's really easy to change the colour but overriding in the strings.xml file. Otherwise if we want to change the colour for one item (and not affect other items), we will have to override the whole layout file.